server/database/queries/eventDelete.js

```diff
@@ -0,0 +1,10 @@
+const dbconniction = require("../db_connection");
+
+const eventDelete = (eventId) => {
+  const sql = {
+    text: "DELETE FROM events WHERE event_id = $1 ",
+    values: [eventId],
+  };
+  return dbconniction.query(sql).catch((err) => console.log(err));
```

Comment:
We talked about this, in queries should not use promise.
you can write return dbconniction.query(sql) and that will be enough